source/hardware/controllers/denon_mc7000.rst

```diff
@@ -265,8 +275,23 @@ Roll Mode
    :figclass: pretty-figures
 
 
-Beatjump Mode
--------------
+Fixed Loop Mode (yellow LED)
+----------------------------
+
+| This mode lets you set a dedicated loop from current position.
+| Another push on the Pad button will turn the loop off.
```

Comment:
```suggestion
This mode lets you set a dedicated loop from current position.
Another push on the Pad button will turn the loop off.
```

Reply:
I'd like to keep line breaks in all those instances where I used "|"